sources_test.go

```diff
@@ -26,6 +27,42 @@ func TestFile(t *testing.T) {
 	}
 }
 
+func TestListFilesMultipleFiles(t *testing.T) {
+	t.Parallel()
+	dir := "testdata/multiple_files"
+	want := fmt.Sprintf("%s/1.txt\n%s/2.txt\n%s/3.tar.zip", dir, dir, dir)
```

Comment:
This is a little confusing; let's write it out in full.

Reply:
Could you please explain what do you mean by full?